### ISSUE TITLE
ApacheBench can not run with Kagou

### ISSUE THREAD

**Issue by serihiro** [serihiro/kagou#3]
Somehow timeout..

```
» ab -v 10 -n 10 'http://127.0.0.1:5006/test.html'
This is ApacheBench, Version 2.3 <$Revision: 1706008 $>
Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
Licensed to The Apache Software Foundation, http://www.apache.org/

Benchmarking 127.0.0.1 (be patient)...INFO: GET header ==
---
GET /test.html HTTP/1.0
Host: 127.0.0.1:5006
User-Agent: ApacheBench/2.3
Accept: */*


---
LOG: header received:
HTTP/1.1 200 OK
Date: Sun, 02 Oct 2016 10:18:39 GMT
Server: kagou
Connection: close
Content-type: text/html
Content-length: 358

<!DOCTYPE html>
<html>
<head>
<meta charset="UTF-8">
<script src="test.js"></script>
<link rel="stylesheet" href="test.css">
<title>Hello, World!</title>
</head>
<body>
<h1>This is Kagou test page</h1>
<p>Hello, World!</p>
<p>こんにちは、世界！</p>
<p>
<a href="test.csv">test.csv</a>
</p>
<p>
<a href="test.json">test.json</a>
</p>
</body>
</html>

LOG: Response code = 200
apr_pollset_poll: The timeout specified has expired (70007)
```

**Comment by serihiro** [comment]
Maybe kagou sends too long packets..

### Wireshark view

#### nginx
![screen shot 2016-10-02 at 10 56 25](https://cloud.githubusercontent.com/assets/681496/19018069/ee7f2e6e-888e-11e6-8b1e-d0bbe52debfd.png)

#### kagou
![screen shot 2016-10-02 at 10 54 44](https://cloud.githubusercontent.com/assets/681496/19018066/b2f4e4d8-888e-11e6-894c-04ff9ff48e4f.png)

**Comment by serihiro** [comment]
Fixed by this commit b4990494ede7ee85241b68dc88399d2916597211

The cause of this is not to send `FIN` packet.
So modify to close after sending header and body.

```
» ab  -c 5 -n 100 'http://127.0.0.1:5006/test.html'
This is ApacheBench, Version 2.3 <$Revision: 1706008 $>
Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
Licensed to The Apache Software Foundation, http://www.apache.org/

Benchmarking 127.0.0.1 (be patient).....done


Server Software:        kagou
Server Hostname:        127.0.0.1
Server Port:            5006

Document Path:          /test.html
Document Length:        358 bytes

Concurrency Level:      5
Time taken for tests:   0.075 seconds
Complete requests:      100
Failed requests:        0
Total transferred:      49400 bytes
HTML transferred:       35800 bytes
Requests per second:    1338.35 [#/sec] (mean)
Time per request:       3.736 [ms] (mean)
Time per request:       0.747 [ms] (mean, across all concurrent requests)
Transfer rate:          645.65 [Kbytes/sec] received

Connection Times (ms)
              min  mean[+/-sd] median   max
Connect:        0    0   0.1      0       1
Processing:     1    3   0.5      3       5
Waiting:        1    3   0.5      3       5
Total:          2    3   0.5      3       5

Percentage of the requests served within a certain time (ms)
  50%      3
  66%      3
  75%      4
  80%      4
  90%      4
  95%      5
  98%      5
  99%      5
 100%      5 (longest request)
```

**Comment by serihiro** [comment]
Todo: Optimization size of send data in once.
